internal/middleware/account_test.go

```diff
@@ -1,15 +1,15 @@
-package middleware_test
+package middleware
```

Comment:
I'd love to keep this and not discuss usage of `_test` in this PR, there is a ticket where we can discuss usage of this https://issues.redhat.com/browse/HMSPROV-163

Reply:
I am good with this, I only requested this because I thought for a moment this is causing the issue.